scripts/weekly-tileset-generation.py

```diff
@@ -28,26 +25,20 @@ def main():
     ascending_frames = get_frames_for_direction('ASCENDING')
     descending_frames = get_frames_for_direction('DESCENDING')
 
-    # frame 21518 has disjoint temporal coverage
-    if 21518 in ascending_frames:
-        ascending_frames.remove(21518)
-
-    HyP3 = sdk.HyP3('https://hyp3-opera-disp-sandbox.asf.alaska.edu/')
+    HyP3 = sdk.HyP3('https://hyp3-api.asf.alaska.edu/')
 
     prepared_jobs = [
-        build_job('desc/vel', 'velocity', descending_frames),
-        build_job('asc/vel', 'velocity', ascending_frames),
+        build_job('desc/vel', descending_frames),
+        build_job('asc/vel', ascending_frames),
     ]
 
     jobs = HyP3.submit_prepared_jobs(prepared_jobs)
     for job in jobs:
-        print(f'https://hyp3-opera-disp-sandbox.asf.alaska.edu/jobs/{job.job_id}')
+        print(f'https://hyp3-api.asf.alaska.edu/jobs/{job.job_id}')
     jobs = HyP3.watch(jobs, timeout=21600, interval=120)
 
     for job in jobs:
-        response = requests.get(
-            f'https://hyp3-opera-disp-sandbox-contentbucket-ibxz8lcpdo59.s3.us-west-2.amazonaws.com/{job.job_id}/tms/openlayers.html'
-        )
+        response = requests.get(f'https://d3gm2hf49xd6jj.cloudfront.net/{job.job_id}/tms/openlayers.html')
```

Comment:
What bucket is this?

Reply:
`https://d3gm2hf49xd6jj.cloudfront.net` fronts `s3://asf-services-web-content-prod`